src/components/shared/upload/Upload.tsx

```diff
@@ -253,11 +253,11 @@ export default function Upload(props: UploadProps) {
         if (findProjectName) setIsProjectTitleDuplicate(true);
         else setIsProjectTitleDuplicate(false);
     }
-
+    useWebsocket(CurrentPage.PROJECTS, handleWebsocketNotification, null, CurrentPageSubKey.FILE_UPLOAD);
```

Comment:
The question is probably more backend related, why we get global notifications for the snapshot imports and project id specific notifications for the normal file uploads. For both cases, the project Id will be already be created at the point of notification received (maybe not?).
If we process global notifications, every other user subscribing to this global notifications would also execute the handlewebsocketnotification function on new messages. All users active in the upload component will receive this notifications independent from any project. At the same time the handle function is intended specifically to the project uploaded by the original user but not for all other users it will also be triggered (with prob undefined projectId, resulting in potential side effects with the resetting, deleting handling of results).